src/angular/schematics/ng-add/migrations/secondary-entry-points-migration.ts

```diff
@@ -1,13 +1,10 @@
 import { DevkitContext, Migration, TargetVersion } from '@angular/cdk/schematics';
+import { EOL } from 'os';
 import * as ts from 'typescript';
 
 import { classNames } from '../data';
 import { sbbAngularModuleSpecifiers } from '../typescript/module-specifiers';
 
-const ONLY_SUBPACKAGE_FAILURE_STR =
```

Comment:
Can you explain me, what this has done before and why this is not used anymore?

Reply:
From what I understood, this was a warning for empty imports. Since I changed it in a way that aggregates the imports and removes all imports from `@sbb-esta/angular-{business, core, public}`, this should no longer affect us.